src/prometheus-stack/chart/templates/uds-package.yaml

```diff
@@ -65,3 +65,18 @@ spec:
         port: 9090
         description: "Grafana Metrics Queries"
 
+      # Custom rules for unanticipated scenarios
+      {{- range .Values.additionalNetworkAllow }}
+      - direction: {{ .direction }}
+        selector:
+          {{ .selector | toYaml | nindent 10 }}
+        {{- if not .remoteGenerated }}
+        remoteNamespace: {{ .remoteNamespace }}
+        remoteSelector:
+          {{ .remoteSelector | toYaml | nindent 10 }}
+        port: {{ .port }}
+        {{- else }}
+        remoteGenerated: {{ .remoteGenerated }}
+        {{- end }}
+        description: {{ .description }}
+      {{- end }}
```

Comment:
Wondering if we might want to support `remoteCidr` here as well? Recognize we don't have this one on the Vector netpol either - might just be easier to allow anything here and `toYaml` it so we don't have to maintain when new options pop up here (in the future we might have something like `remoteHost`). Thoughts?

Reply:
fixed this in latest commit